lua/zk/pickers/minipick.lua

```diff
@@ -0,0 +1,73 @@
+local M = {}
+local H = {}
+local minipick = require("mini.pick")
+
+-- Picker from zk-nvim must implement this.
```

Comment:
Tbh I don't really like that comment. The other picker implementations don't have "interface implementation" commentend either and I don't really see any added value in them.

Reply:
Agreed. That was a comment I added to show the maintainer of mini how I integrated it here. I’ll remove it.